diff -urN aMule-2.3.1rc2/configure.in aMule-2.3.1rc2-patched/configure.in
--- aMule-2.3.1rc2/configure.in	2011-08-17 08:25:07.000000000 +1100
+++ aMule-2.3.1rc2-patched/configure.in	2011-10-04 19:50:42.083054127 +1100
@@ -153,7 +153,7 @@
 	MULE_IF_ENABLED_ANY([monolithic, amule-daemon, amule-gui, amulecmd, webserver, fileview], [WX_UNICODE=1])
 	DEBUG=$WX_DEBUG
 	WX_CONVERT_STANDARD_OPTIONS_TO_WXCONFIG_FLAGS
-	WX_CONFIG_CHECK([2.8.12], [wxWin=1],,, [$WXCONFIG_FLAGS])
+	WX_CONFIG_CHECK([2.8.11], [wxWin=1],,, [$WXCONFIG_FLAGS])
 	AS_IF([test "$wxWin" != 1],
 		[AC_MSG_ERROR([
 		wxWidgets must be installed on your system but wx-config 
